doc/website/getting-started/what-is-iceoryx.md

```diff
@@ -1 +1,24 @@
 # What is Eclipse iceoryx?
```

Comment:
Please break the lines after 120 characters. A markdown linter will help your here.